pkg/http/handler/handler.go

```diff
@@ -69,28 +86,275 @@ func (h HTTPHandler) reportError(w http.ResponseWriter, status int, err error) {
 func (h HTTPHandler) GetRouter() *mux.Router {
 	router := mux.NewRouter()
 
-	router.HandleFunc("/items", h.ListItems).Methods("GET")
+	router.HandleFunc("/users", h.AddUser).Methods("POST")
+	router.HandleFunc("/users/{id}", h.GetUser).Methods("GET")
+	router.HandleFunc("/users", h.ListUsers).Methods("GET")
+	router.HandleFunc("/users/{id}", h.UpdateUser).Methods("PUT")
+	router.HandleFunc("/users/{id}", h.DeleteUser).Methods("DELETE")
+
+	router.HandleFunc("/accounts", h.AddAccount).Methods("POST")
+	router.HandleFunc("/accounts/{id}", h.GetAccount).Methods("GET")
+	router.HandleFunc("/accounts/{id}", h.UpdateAccount).Methods("PUT")
+	router.HandleFunc("/accounts/{id}", h.DeleteAccount).Methods("DELETE")
+	router.HandleFunc("/accounts", h.ListAccounts).Methods("GET")
```

Comment:
Can split this part with sub routers if too many endpoints.

Reply:
Or create 2 functions with endpoints(`GetAccountRouter` and `GetUserRouter`)

Reply:
Have no idea why is it too many. IMO looks OK.